dataeng/jobs/analytics/Enrollment.groovy

```diff
@@ -31,6 +27,26 @@ class Enrollment {
                         }
                     }
                 }
+                parameters common_parameters(allVars, env_config)
+                parameters from_date_interval_parameter(allVars)
+                parameters to_date_interval_parameter(allVars)
+                parameters {
+                    stringParam('BASE_OUTPUT_URL', env_config.get('BASE_OUTPUT_URL', allVars.get('BASE_OUTPUT_URL')), '')
+                    stringParam('OUTPUT_DIR', env_config.get('OUTPUT_DIR', allVars.get('OUTPUT_DIR')), '')
+                    stringParam('OUTPUT_SUFFIX', env_config.get('OUTPUT_SUFFIX', allVars.get('OUTPUT_SUFFIX')), '')
+                    stringParam('CREDENTIALS', env_config.get('CREDENTIALS', allVars.get('CREDENTIALS')), '')
+                }
```

Comment:
What's up with all these new parameters?  Seems unrelated to heartbeats?

Reply:
Ugh - smh. This is copy pasta from another job. I'll remove it.

Reply:
Nice catch!

Reply:
Removed now.